tests/platform_tests/api/test_component.py

```diff
@@ -9,6 +9,17 @@
 
 from platform_api_test_base import PlatformApiTestBase
 
+###################################################
+# TODO: Remove this after we transition to Python 3
+import sys
+if sys.version_info.major == 3:
+    STRING_TYPE = str
+else:
+    STRING_TYPE = basestring
+# END Remove this after we transition to Python 3
+###################################################
+
+
```

Comment:
Just a generic comment, should we keep this logic in all the files modified,(line 12-20) 
or if we could separate this part in a separate file example platform_api_test_base, thats ways eventually it would be easier to remove this part ?

Reply:
For a permanent code change, I would agree with this 100%, as it eliminates duplicate code. But seeing as this is a temporary workaround until we move to Python 3, having it in each of the files which require it makes it a.) more visible and b.) more isolated.